protein_ligand/README.md

```diff
@@ -102,11 +102,25 @@ If multiple submissions are incorrectly provided as "ranked" by a single partici
 
 ## Stage 2: Prediction of binding poses for binding fragment
 
-Plans for stage 2 are still being finalized, but this is planned to involve predicting the bound structures of the compounds which bind, the identity of which will be released at the end of the first stage.
+### Setup and description of Stage 2
+
+**Aim**: The second part of the SAMPL7 challenge builds onto the first stage. The objective is to correctly predict the binding pose for each of protein-fragment complex identified by [PanDDA](https://pandda.bitbucket.io/). In total, 52 fragments were found to bind to the C2 crystal form described in stage 1. 45 hits were identified at the first site (denoted by a helium atom `S1` in the provided structure (`PHIPA_C2_apo_sites.pdb`)). 4 were found at the second site (denoted by a neon atom `S2`). The third (denoted by an argon atom `S3`) and the fourth site (denoted by a krypton atom `S4`) were each found to bind to a single distinct fragment. Accurate binding pose prediction is essential in structure-based drug discovery and in the elaboration/ optimization of known drugs and chemical probes.   
```

Comment:
It's odd to have the "accurate binding pose prediction" sentence in this paragraph since it seems to have nothing to do with most of the rest of this paragraph. Maybe it should be merged into the second sentence, so "The objective is to ... identified by PanDDA, partly because accurate binding pose prediction is seen as essential for structure-based drug design."

Otherwise just move it somewhere else, I guess.